docs/configuration.rst

```diff
@@ -290,9 +318,10 @@ These settings can be used to specify where to find other settings through custo
 
     .. warning::
         When this setting is defined, all other combinations of :envvar:`MAGPIE_CONFIG_DIR`,
-        :envvar:`MAGPIE_PERMISSIONS_CONFIG_PATH` and :envvar:`MAGPIE_PROVIDERS_CONFIG_PATH` are effectively
-        ignored in favour of definitions in this file. It is not possible to employ the :ref:`config_file` at
-        the same time as multi-configuration loading from a directory.
+        :envvar:`MAGPIE_PERMISSIONS_CONFIG_PATH`, :envvar:`MAGPIE_PROVIDERS_CONFIG_PATH` and
+        :envvar:`MAGPIE_WEBHOOKS_CONFIG_PATH` are effectively ignored in favour of definitions in this file.
```

Comment:
Nice to have this specified in the config. 
Do you think it would be helpful to also have a warning in case one of these envvars are non-null while having `MAGPIE_CONFIG_PATH` defined?

Reply:
Yeah, we can add a log entry that says var "x" not used because `MAGPIE_CONFIG_PATH` was detected.